src/decompressor.rs

```diff
@@ -0,0 +1,116 @@
+use std::ffi::OsStr;
+use std::io::Read;
+use std::path::Path;
+use std::collections::HashMap;
+use std::process::Command;
+use std::process::Stdio;
+use std::process::ChildStdout;
+use globset::{Glob, GlobSet, GlobSetBuilder};
```

Comment:
Imports should be consistent with the rest of the code. They should be grouped in three blocks, where each block is separated by whitespace. The first block are `std` imports. The second block are external crate imports. The third block are internal crate imports. Each block should be sorted lexicographically.